cmd/chartsvc/handler.go

```diff
@@ -200,6 +200,35 @@ func getChartVersionValues(w http.ResponseWriter, req *http.Request, params Para
 	w.Write([]byte(files.Values))
 }
 
+func findLatestChart(name, version, appversion string) []*models.Chart {
```

Comment:
This isn't really finding the latest chart more than just finding the chart with filters, right? I believe this/or the handler should be called `listChartsWithFilters` or similar. Although I'm still not sure we need a separate function for this here, given we mock the HTTP requests everywhere else.

Reply:
I still think that having atomic functions that don't deal with different things is a slightly better approach but since the rest of methods follow that approach I am okay merging the methods.